tests/bsim/bluetooth/ll/bis/src/main.c

```diff
@@ -311,6 +311,8 @@ static void create_big(struct bt_le_ext_adv *adv, struct bt_iso_big **big)
 	int err;
 
 	printk("Creating BIG...\n");
+	is_iso_connected = false;
+	bis_channels[0]->iso = NULL;
```

Comment:
Suggest to add a comment that `bis_channels[0]->iso = NULL;` is necessary due a bug in the host. 

The `bis_channels[0]->iso` value should never be directly assigned by the application